gef.py

```diff
@@ -6582,24 +6602,26 @@ class GlibcHeapChunkCommand(GenericCommand):
     See https://github.com/sploitfun/lsploits/blob/master/glibc/malloc/malloc.c#L1123."""
 
     _cmdline_ = "heap chunk"
-    _syntax_  = "{:s} LOCATION".format(_cmdline_)
+    _syntax_  = "{:s} [-h] [--allow-unaligned] LOCATION".format(_cmdline_)
 
     def __init__(self):
         super().__init__(complete=gdb.COMPLETE_LOCATION)
         return
 
+    @parse_arguments({"LOCATION": ""}, {"--allow-unaligned": True})
```

Comment:
Please use `address` for consistency

Reply:
ok, I understand. I was thinking about gdb's difference between a location and an address when choosing "LOCATION" . I will change it.